tsdb/index.go

```diff
@@ -9,6 +9,7 @@ import (
 	"sort"
 	"sync"
 
+	"github.com/RoaringBitmap/roaring"
```

Comment:
Can we not dip into the implementation of `SeriesIDSet`? If we need something from the `roaring.Bitmap` we could maybe just add it to `SeriesIDSet`? It will make future optimisations to the `SeriesIDSet` a bit easier.

Reply:
Fixed in 979d790.